__tests__/app.test.js

```diff
@@ -163,4 +163,61 @@ describe('GET /api/reviews/:review_id/comments', () => {
       expect(response.body.msg).toBe('bad request')
       })
   });
+});
+
+describe("/api/reviews/:review_id/comments", () => {
+  test("Get status 201 response", () => {
+  const newComment = {
+        username: "mallionaire",
+        body: "giant Jenga is better",
+      };
+      return request(app)
+        .post("/api/reviews/2/comments")
+        .send(newComment)
+        .expect(201)
+  })  
+  test("Get status 201 response and an object containing the newly posted comment", () => {
+      const newComment = {
+        username: "mallionaire",
+        body: "giant Jenga is better",
+      };
+      return request(app)
+        .post("/api/reviews/2/comments")
+        .send(newComment)
+        .expect(201)
+        .then((response) => {
+          expect(response.body.comment.comment_id).toBe(7);
+          expect(response.body.comment.body).toBe("giant Jenga is better");
+          expect(response.body.comment.review_id).toBe(2);
+          expect(response.body.comment.author).toBe("mallionaire");
+          expect(response.body.comment.votes).toBe(0);
+          expect(typeof response.body.comment.created_at).toBe("string");
+          });
+  });
+  test("Return error 404 and message of 'no review found at this id!'", () => {
+   const newComment = {
+        username: "be-part",
+        body: "giant Jenga is better",
+      };
+  return request(app)
+  .post("/api/reviews/20/comments")
+        .send(newComment)
+        .expect(404)
+        .then((response) => {
+         expect(response.body.msg).toBe('no review found at this id!')
+        })
+  });
+  test("Return error 400 and message of 'bad request'", () => {
+   const newComment = {
+        username: "be-part",
+        body: "giant Jenga is better",
+      };
+  return request(app)
+  .post("/api/reviews/NotAnId/comments")
+        .send(newComment)
+        .expect(400)
+        .then((response) => {
+         expect(response.body.msg).toBe('bad request')
+        })
+  });
```

Comment:
Great tests so far, however there are some cases which have been be tested for yet. 

What should happen if a request was made that was miss some of the required fields in the request body? i.e. no username or body properties.

What should happen if the username given in the request body did not exist?

What should happen if extra properties were given in the req body?